LiteCore/Database/SequenceTracker.cc

```diff
@@ -475,9 +501,9 @@ namespace litecore {
 #pragma mark - DOC CHANGE NOTIFIER:
 
 
-    DocChangeNotifier::DocChangeNotifier(SequenceTracker &t, slice docID, Callback cb)
+    DocChangeNotifier::DocChangeNotifier(SequenceTracker &t, CollectionSpec collection, slice docID, Callback cb)
```

Comment:
same here, the invariant is t->collection == collection. We don't need the collection argument if it is readily fetched from SequenceTracker.